doc/source/ray-air/doc_code/tf_starter.py

```diff
@@ -70,7 +70,9 @@ def to_tensor_iterator():
 
     results = []
     for _ in range(epochs):
-        tf_dataset = to_tf_dataset(dataset=dataset, batch_size=batch_size)
+        tf_dataset = to_tf_dataset(dataset=dataset, batch_size=batch_size).prefetch(
+            tf.data.AUTOTUNE
+        )
```

Comment:
Minor point, but would prefer to have all dataset operations contained inside the `to_tf_dataset` method.